### PR TITLE
Update tile guidelines

### PR DESCRIPTION
**Pull request by blackbaud-johnly**
Addresses https://github.com/blackbaud/skyux2-docs/issues/883 and https://github.com/blackbaud/skyux2-docs/issues/892.

### CODE REVIEW

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-tile-guidelines-04020](https://developer.blackbaud.com/skyux?_sp.spa=pr-tile-guidelines-04020)

File: src/app/components/tile/index.html [hunk 1]
```diff
@@ -214,9 +214,11 @@
     <p>
       Tiles are containers to organize varied content. They divide pages into multiple categories and group related content. Use tiles to provide access to a wide range of content where users benefit from the ability to reorder, expand and collapse, and personalize tiles.
     </p>
-
+    <p>
+      The content within tiles can vary widely, but each tile follows <a routerLink="/design/principles">SKY UX design principles</a> and <a routerLink="/design/guidelines">design guidelines</a> for that typography, layout, and other design considerations.
```
Comment (Blackbaud-ToddRoberts) [inline src/app/components/tile/index.html:218]: last part of the sentence reads a bit awkwardly, maybe something like "for layout, typography and other design consuderations."

Comment (Blackbaud-DeniseCisneros) [reply src/app/components/tile/index.html:218]: I agree with Todd, but I don't really have a preference for order of items. I'd just eliminate the word "that" before typography.

Comment (blackbaud-johnly) [reply src/app/components/tile/index.html:218]: Yep. `that` definitely shouldn't be there. It's just a leftover word that didn't get deleted when I was tweaking the sentence. I'll remove it and flip the order like Todd suggests.

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-tile-guidelines-04020](https://developer.blackbaud.com/skyux?_sp.spa=pr-tile-guidelines-04020)

File: src/app/components/tile/index.html [hunk 2]
```diff
@@ -214,9 +193,11 @@
     <p>
       Tiles are containers to organize varied content. They divide pages into multiple categories and group related content. Use tiles to provide access to a wide range of content where users benefit from the ability to reorder, expand and collapse, and personalize tiles.
     </p>
-
+    <p>
+      The content within tiles can vary widely, but each tile follows <a routerLink="/design/principles">SKY UX design principles</a> and <a routerLink="/design/guidelines">design guidelines</a> for layout typography, and other design considerations.
```
Comment (Blackbaud-ToddRoberts) [inline src/app/components/tile/index.html:197]: missing a comma between layout & typography

Comment (blackbaud-johnly) [reply src/app/components/tile/index.html]: Updated. Thanks!

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-tile-guidelines-04020](https://developer.blackbaud.com/skyux?_sp.spa=pr-tile-guidelines-04020)

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-tile-guidelines-04020](https://developer.blackbaud.com/skyux?_sp.spa=pr-tile-guidelines-04020)

**Pull request merged**